### ISSUE TITLE
JNI doesn't support memtable_whole_key_filtering

### ISSUE THREAD

**Issue by jokea** [facebook/rocksdb#8614]
Hi folks,

I tried to add bloom filter for memtable, and found out jni doesn't support this option: memtable_whole_key_filtering.
Are there any other options that can do the job or this feature is not supported?

Thanks,
Jokea

**Comment by ltamasi** [comment]
Right, it doesn't seem to be exposed via the Java API. I don't think it would be hard to add but Cc @adamretter just in case
